README.md

```diff
@@ -3,7 +3,7 @@
 > The leaderboard website displays scores submitted by different players. It also allows you to submit your score. All data is preserved thanks to the external Leaderboard API service.
 ## Live Demo
 
-[Live Demo Link](link here)
```

Comment:
It looks like you forgot to add the MIT license to your project. Kindly fix this issue as well.